### PR TITLE
Column width storybook

### PR DESCRIPTION
**Pull request by atmgrifter00**
# Pull Request

## 🤨 Rationale

- #1044

## 👩‍💻 Implementation

Adding storybook for configuring column `fractionalWidth` and `minimumPixelWidth` following pattern put in place by @jattasNI.

### CODE REVIEW

File: packages/nimble-components/src/table-column/base/tests/table-column.stories.ts [hunk 1]
```diff
@@ -456,3 +456,142 @@ export const sorting: StoryObj<SortingTableArgs> = {
...
+        },
+        {
+            columnId: 'quote-column',
+            width: 2
+        }
+    ]
+} as const;
+
+const fractionalWidthDescription = `Some table columns (like \`nimble-table-column-text\`) can be configured to have a fractional width such that its width
+                                    will be proportional to other columns also configured with a fractional width (which is the default). So,
```
Comment (mollykreis) [inline packages/nimble-components/src/table-column/base/tests/table-column.stories.ts:496]: > (which is the default)

This isn't necessarily true from a column API perspective, right? A given column will decide what is the default and whether it even supports both fractional & fixed sizing, right?

Comment (atmgrifter00) [reply packages/nimble-components/src/table-column/base/tests/table-column.stories.ts:496]: Yeah, I can see the confusion. I think what I'm trying to convey here, is that even columns that don't expose the `fractional-width` API to control their fractional-width, will still be sized in response to a table size (or by another column changing its `fractional-width`) in a way that mirrors it having a `fractional-width` of 1.

Comment (atmgrifter00) [reply packages/nimble-components/src/table-column/base/tests/table-column.stories.ts]: Updated comment.

File: packages/nimble-components/src/table-column/base/tests/table-column.stories.ts [hunk 2]
```diff
@@ -456,3 +456,144 @@ export const sorting: StoryObj<SortingTableArgs> = {
...
+            width: 2
+        }
+    ]
+} as const;
+
+const fractionalWidthDescription = `Some table columns (like \`nimble-table-column-text\`) can be configured to have a fractional width such that its width
+                                    will be proportional to other columns also configured with a fractional width. So, a column with a \`fractional-width\`
+                                    set to 2 would be twice as wide as a column with a \`fractional-width\` set to 1. The default value for
+                                    \`fractional-width\` is 1. Columns that don't support \`fractional-width\` explicitly, or another API responsible for
+                                    managing the width of the column, will behave as if they have a \`fractional-width\` of 1.`;
```
Comment (fredvisser) [inline packages/nimble-components/src/table-column/base/tests/table-column.stories.ts:499]: I'm probably slow, but I had to read this several times to understand – perhaps:

```suggestion
const fractionalWidthDescription = `Configure each column's width relative to the other columns with the \`fractional-width\` property. For example, a column with a \`fractional-width\` set to 2 will be twice as wide as a column with a \`fractional-width\` set to 1. 
The default value for \`fractional-width\` is 1, and columns that don't support \`fractional-width\` explicitly, or another API responsible for managing the width of the column, will also behave as if they have a \`fractional-width\` of 1.`;
```

**Review by fredvisser — APPROVED** [review]

**Review by jattasNI — APPROVED** [review]

**Pull request merged**